src/briefcase/commands/build.py

```diff
@@ -86,6 +93,7 @@ def _build_app(
     def __call__(
         self,
         app: AppConfig | None = None,
+        app_name: str | None = None,  # New argument for filtering
```

Comment:
There's no need for a comment like this one. It may be a new argument *in this PR*, but as soon as it's merged to main, it won't be a "new" argument any more.